docs/API/LorisRESTAPI_v0.0.3.md

```diff
@@ -814,3 +814,49 @@ files, and a `.tar.gz` of the raw DICOM data as acquired during the candidate
 scanning session, and as retrieved from `/candidates/$CandID/$Visit/dicoms`.
 
 Only `GET` is currently supported.
+
+### 5.3 Upload DICOM file
+```
+PUT /candidates/$CandID/$VisitLabel/dicoms/$Filename/$IsPhantom
+```
+
+Upload a DICOM file that is compressed as a `.zip`,`.tar.gz` or `.tgz`.
+$CandID must be the full name of the file and its extension.
```

Comment:
I assume this is a typo and you meant '$Filename' not "$CandID"?

Reply:
A typo indeed!